README.md

```diff
@@ -1,17 +1,86 @@
 # Google AI Swift SDK
 
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fgoogle%2Fgenerative-ai-swift%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/google/generative-ai-swift)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fgoogle%2Fgenerative-ai-swift%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/google/generative-ai-swift)
```

Comment:
I don't think these are accurate, I'd rather wait until they're updated appropriately. Right now it shows Swift 5.7+ which I believe we're only 5.9+, and it says visionOS, Linux, watchOS are supported which they are not (untested at least)

Reply:
OK. Removed for now